pootle/static/css/auth.css

```diff
@@ -63,9 +63,7 @@
     box-sizing: border-box;
     padding: 1em;
     width: 50%;
-
```

Comment:
hmmm such empty lines have been intentional to split blocks of similar declarations. Probably worth checking the [`declaration-block-properties-order`](http://stylelint.io/user-guide/rules/declaration-block-properties-order/) rule and set groups there, although I understand that might become a bit noisy.